Slider/Slider.jsx

```diff
@@ -66,6 +66,11 @@ export default class Slider extends MaterialComponent {
           <div class="mdc-slider__track" />
         </div>
         <div class="mdc-slider__thumb-container">
+          {allprops.discrete && (
```

Comment:
just read this from `props` instead of `allprops`

Reply:
I changed that to props, but now i am thinking, should not be destructed allprops like tabindex?